MANIFEST.in

```diff
@@ -1,8 +1,23 @@
-#include *.txt
+include AUTHORS.md
+include LICENSE
+include README.md
+
 include versioneer.py
 include pvlib/_version.py
+
 recursive-include pvlib/data *
-include README.md
+
 include pvlib/spa_c_files/*.h
 
-#recursive-include docs *.txt
+recursive-exclude * __pycache__
+recursive-exclude * *.py[co]
+
+recursive-include docs *
+prune docs/tutorials
```

Comment:
just curious, why prune the Jupyter notebook tutorials? IMO these could be run locally more interactively, than viewing them in nbviewer or GitHub. IMO pvlib should include these in the sdist.

Reply:
Yes, better to include them.